### ISSUE TITLE
Start and stop autoplay programatically

### ISSUE THREAD

**Issue by jpolack** [leecade/react-native-swiper#225]
Hello everyone,

is it yet possible to start and stop autoplay programatically?

If not this might be a nice feature.

Thanks

**Comment by ermankuruoglu** [comment]
Same ...

**Comment by ermankuruoglu** [comment]
Hi,

You can bind swiper autoplay property your screen state on below.

<Swiper
  autoplay={this.state.swiperAutoPlay}
>

If you want to stop swiper at any time, just set the swiperAutoPlay value to false.

Good luck.

**Comment by autume** [comment]
@ermankuruoglu this method can stop autoplay，but can not start autoplay after stop

**Comment by asgvard** [comment]
Yes, it only works one way, because it kills the timer if the new props are "!autoplay". But it sets the timer only onScrollEnd, which never occurs automatically after it's stopped.

**Comment by ifours** [comment]
Meanwhile you can use `<Swiper ref={node => this.swiper = node} />`. And after that `this.swiper.autoplay()`

**Comment by romit07** [comment]
> Meanwhile you can use `<Swiper ref={node => this.swiper = node} />`. And after that `this.swiper.autoplay()`

Does this solution works? It's not working in my case

**Comment by Osmon11** [comment]
just changed my autoplay state to null and vice versa
